app/server/services/sendgrid_email.js

```diff
@@ -0,0 +1,130 @@
+var moment = require('moment');
+
+var path = require('path');
+
+var templatesDir = path.join(__dirname, '../templates');
+var emailTemplates = require('email-templates');
+
+var ROOT_URL = process.env.ROOT_URL;
+
+var ACCEPTANCE_TEMPLATE = process.env.ACCEPTANCE_TEMPLATE;
+var SENDGRID_API_KEY = process.env.SENDGRID_API_KEY;
+var HACKATHON_NAME = process.env.HACKATHON_NAME;
+var EMAIL_ADDRESS = process.env.EMAIL_ADDRESS;
+var TWITTER_HANDLE = process.env.TWITTER_HANDLE;
+var FACEBOOK_HANDLE = process.env.FACEBOOK_HANDLE;
+
+var SMTP_CLIENT_ID = process.env.SMTP_CLIENT_ID;
+var SMTP_CLIENT_SECRET = process.env.SMTP_CLIENT_SECRET;
+var SMTP_CLIENT_REFRESH = process.env.SMTP_CLIENT_REFRESH;
+
+var ACCEPTANCE_EMAIL_TEMPLATE = process.env.ACCEPTANCE_TEMPLATE;
+var VERIFICATION_EMAIL_TEMPLATE = process.env.VERIFICATION_EMAIL_TEMPLATE;
+var PASSWORD_RESET_EMAIL_TEMPLATE = process.env.PASSWORD_RESET_EMAIL_TEMPLATE;
+var PASSWORD_CHANGED_EMAIL_TEMPLATE = process.env.PASSWORD_CHANGED_EMAIL_TEMPLATE;
+var WAIVER_EMAIL_TEMPLATE = process.env.WAIVER_EMAIL_TEMPLATE;
+
+var EMAIL_CONTACT = process.env.EMAIL_CONTACT;
+/*var EMAIL_HEADER_IMAGE = ';
+if(EMAIL_HEADER_IMAGE.indexOf("https") == -1){
+  EMAIL_HEADER_IMAGE = ROOT_URL + EMAIL_HEADER_IMAGE;
+}
+*/
+var NODE_ENV = process.env.NODE_ENV;
+
+
+
+const sgMail = require('@sendgrid/mail');
+sgMail.setApiKey(SENDGRID_API_KEY);
+
+var controller = {};
+
+function sendOne(template, emails, templateData, callback) {
```

Comment:
Add the @params to define what the param types should be. 
Also change emails to email since this is sendOne